### PR TITLE
Update pre-commit

### PR DESCRIPTION
**Pull request by MRegeard**
This PR updates pre-commit with new version 0.7.2.
I also added `"--fix"` and `"--show-fixes"` to arguments.

### CODE REVIEW

**Review by registerrier — APPROVED** [review]
Thanks @MRegeard . Looks good to me.

**Review by morcuended — DISMISSED** [review]
Looks good! Just a comment on the latest version available. I wonder if this update can be spotted by the dependant bot.

EDIT: It can be done with a `pre-commit.ci` file.

**Comment by MRegeard** [comment]
@morcuended Thanks.
I made a request to link pre-commit.ci to our repo.

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,13 +1,27 @@
...
+      - id: check-ast
+      - id: check-case-conflict
+      - id: trailing-whitespace
+      - id: check-yaml
+      - id: debug-statements
+      - id: check-added-large-files
+        args: ["--enforce-all", "--maxkb=1054"]
+        exclude: ""
+      - id: end-of-file-fixer
+      - id: mixed-line-ending
```
Comment (registerrier) [inline .pre-commit-config.yaml:24]: What does this change w.r.t. current state?

**Review by registerrier — COMMENTED** [review]
Thanks @MRegeard . Moving to pre-commit.ci is probably something we should do. But maybe not for 1.3.
Does this PR require this? If yes, I would postpone to 2.0 milestone.

File: .pre-commit-config.yaml (hunk 1 above)
Comment (MRegeard) [reply .pre-commit-config.yaml:24]: I copied that from another repo because I thought this was triggering pre-commit.ci.

**Comment by MRegeard** [comment]
@registerrier, you're right, for 1.3 let's just update ruff-pre-commit and I'll do another PR for pre-commit.ci !

**Review by registerrier — APPROVED** [review]
Thanks @MRegeard

**Pull request merged**